frontend/public/locales/en/translation.json

```diff
@@ -1194,6 +1194,9 @@
   "discoveryConfig.add": "Create",
   "discoveryConfig.autoImportPolicy.Info": "After discovery settings are configured, discovered Red Hat OpenShift on AWS clusters can be automatically imported as managed clusters using policy enforcement. Policies used in this way can also be configured with filters or annotations to specify which clusters should be automatically imported.",
   "discoveryConfig.cancel": "Cancel",
+  "discoveryConfig.ClusterTypes.label": "Cluster types",
+  "discoveryConfig.ClusterTypes.labelHelp": "All cluster types are included by default unless specified in this drop-down menu.",
```

Comment:
@oafischer Could you help with the wording here? It almost sounds to me like the checked cluster types will not be included. But really, if nothing is checked, then everything is included. Otherwise, it's just the ones the user has selected. I also think we should probably avoid the word "drop-down" or at least remove they hyphen.

My bad here... I recommended @vishsanghishetty use the existing help for the version dropdown, but that's where this unclear wording originated: "All Red Hat OpenShift versions are included by default unless specified in this drop-down menu."

Reply:
Maybe: _By default, all cluster types are included. Use the drop-down menu to select only specific cluster types._

Drop-down is actually the correct word to use here as per the Style guide, though it's tagged as "use with caution"

![image](https://github.com/user-attachments/assets/528669b1-6b38-48a1-92af-2056257639d8)